integration-tests/orm-tests/orm-tests.bats

```diff
@@ -58,6 +62,14 @@ teardown() {
   # https://github.com/prisma/prisma/tree/main/packages/integration-tests/src/__tests__/integration/mysql
 }
 
+@test "TypeORM smoke test" {
```

Comment:
A small description what TypeORM is for like what Peewee and Prisma smoke test have on top of them would be nice.

Reply:
Good suggestion! I'll add those in before I merge.